metadata.json

```diff
@@ -12,15 +12,28 @@
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
         "6",
-        "7"
+        "7",
+        "8"
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "10.04",
         "12.04",
-        "14.04"
+        "14.04",
+        "15.04",
+        "15.10",
+        "16.04",
+        "16.10",
+        "17.04"
+      ]
+    },
+    {
+      "operatingsystem": "Linux Mint",
+      "operatingsystemrelease": [
+        "13",
+        "17*",
+        "18*"
```

Comment:
[metadata-json-lint](https://github.com/voxpupuli/metadata-json-lint) does not give me any warning, however, looking at [docs.puppet.com](https://docs.puppet.com/puppet/latest/modules_metadata.html#specifying-operating-system-compatibility) I agree that this might not be the correct approach.

How about "`17.x`" and "`18.x`", like the semantic versioning example they provide for `puppetlabs/stdlib`?